internal/accounts/accounts.go

```diff
@@ -86,6 +86,13 @@ func (r *accountResult) String() string {
 	var b bytes.Buffer
 	writer := util.CreateTabWriter(&b)
 
+	if r.Address.IsValid(flow.Testnet) {
```

Comment:
Can we base this off of `selectedNetwork` instead? https://github.com/onflow/flow-cli/pull/1326/files#diff-8cfd65a8348c96dd493e6ed1bc0b2c91340dc16c9026b1e7d8d4c71d1d20de5eR53

If we only validate by the codeword, then crescendo and testnet will be the same, so probably best if we differentiate between them (I'm assuming crescendo will eventually be added to the CLI?).